data/json/mapgen/collapsed_tower.json

```diff
@@ -0,0 +1,195 @@
+        "#######~~~~~~~~~~~~~~~~~~~~~~~~~~~##~~~~~~~~###~~~|~~~~~~~~~~~~~~~~~~###",
+        "#####~~~~~~~~~~~~~~~~~~~~~~~~~~~~~#~~~~~~~###~~~~~~~~~~~~~~~~~~~~~~~####",
+        "########################  ######~~##~~~##########|~~~~|##~~ ############",
+        "######################   #####    ###~~~#########|~~~~|## ~ ############",
+        "######################  ###############~~########|~~~~|||  ~||||||||####",
+        "######################  ||||||||||#|####~~#######|~~##|  ~         |####",
+        "#####||||||||||||||||| ||          |#############|~~~#|        6666|####",
+        "#####|xppCCppppCCppp6| #|##  ####  |# ~~~~~######|#~~#|22{{22222222|####",
+        "#####|xppppppppppppp6| #|#  ####  #|# ~#~~~######|####|        6666|####",
+        "#####|222222ppY66ppp6|  |##########|   # ~~~    # ~~~~##           |####",
+        "#####|ppppp2ppppppppp|# |||   ## ##|# ~~#~~~~~~ #~~~#          BB  |####",
+        "#####|p//pp{ppppppppp3# 3##| ##|   {~~~~~~~~~~~~~~~~##||||||BB#||  |####",
+        "#####|ppppp2ppppppppp3  3 #| # +   { ##   ########~~ #|#########   |####",
+        "#####|222222ppY66ppp6| #|##|###|   |||||||||||||||~~~~|###|#   #   |####",
+        "#####|xppppppppppppp6|  |#|||||| ##|##########~~#|##~~|###|#   # ##|####",
+        "#####|xppCCppppppppp6|  |##  ##|   |#########~~~~~####||||||||||###|####",
+        "#####||||||||||#ppppp| #|  # 66|# #|#~~~~~~~~~~~~~~~~#|########### |####",
+        "##############|#2{{22|  +     B||  ||######~~t~~~|       ########  |####",
+        "################ppppr|# |      |         ##~~~~~~|    |#    ####   |####",
+        "#################pppr|33|  ####|###||   ##### ~~~||33||##         #|####",
+        "###############pppppp|##||||###|  ########~~~~~~~|#      #   6#####|####",
+        "################p##pR|# |######|     # ##~~~~####|#           #####|####",
+        "#############|||||||||  |######|    ######~~~####|  ##|#66#########|####",
+        "#####################|  |  ####|       ##~~######| ###||||||||||||||####",
+        "#####################|  | #####|  ###|||   |||||||    3   |#############",
+        "#####################|  3    ##|   ##| 6   |###3~~    3   |#############",
+        "####################||  |#     |     |B6   |###3      |   |#############",
+        "#####||||||||||||||||#  |######|     | 6   |||||      |||||#############",
+        "##### #  |R         |#  |######|||   +     #####      w  ||#############",
+        "#########|#######  #||  |||     ##   +    #|||||     |||||||############",
+        "#########|######  ## #    +      #   |#   #|   3     3   ||#############",
+        "#########|#######   ###   +  # ######|||###|   3     ||||||#############",
+        "#########|########  |||||||  #||||||||#   #|   ||||||###################",
+        "#########|##       R|#####   ########|     ||||||||#####################",
+        "#########||||||||||||####### ########|    ||############################",
```

Comment:
You can't get into this room here, is that intentional?

Reply:
That is intentional, you can sometimes enter the room due, to the random terrain bashes. 
The building as a whole has a few other shortcuts that can happen based on the bashes as well.

Reply:
Ah, okay. I still think that this may need a tiny bit of tweaking, I pulled your branch and explored the building a bit, and this room was blocked off (I couldn't tell it was there), but it had nanomaterial canisters in it.
If you think it's fine that it's possible to totally lock some players out of getting the loot in this room (or knowing it's there), based on rng, that's totally fine, and you can collapse this thing.